pkg/defaultRules/defaultRules.yaml

```diff
@@ -1548,3 +1548,273 @@ rules:
       anyOf:
         - $ref: "#/definitions/metadataNamePattern"
         - $ref: "#/definitions/metadataGenerateNamePattern"
+  - id: 55
+    name: Ensure container probes have an initial delay configured
```

Comment:
I hate to be "that guy" but we called it "ensure each container... has..." in all the other rules:
![image](https://user-images.githubusercontent.com/19731161/175284284-2e7b7dee-b28f-4292-b326-a1349ba87f37.png)